sdk/eidolon_ai_sdk/system/resources/resources_base.py

```diff
@@ -35,12 +36,15 @@ def load_resources(path):
     logger.debug(f"Loading resources from {os.path.abspath(path)}")
     if not os.path.exists(path):
         raise ValueError(f"Path {path} does not exist")
+    resource_path = os.path.dirname(os.path.realpath(path))
+    if resource_path not in sys.path:
+        sys.path.append(resource_path)
```

Comment:
Can you make this an argument? I think in most cases the custom components will not live with the resource definitions.

I am building a template repo that shows how I expect actual usage a machine to look like [machine template](https://github.com/eidolon-ai/agent_machine ).

Reply:
@LukeLalor finally had a moment to look at this: the path is already an argument. The logic now it to assure that this path, whatever it might be, is part of sys.path so that, both eidolon and other packages are guaranteed to find it as needed. I think this is reasonable. Maybe I didn't understand your suggestion clearly?